geode-core/src/main/java/org/apache/geode/management/internal/beans/MemberMBean.java

```diff
@@ -148,7 +148,12 @@ public long getInitialImageTime() {
 
   @Override
   public int getInitialImagesInProgres() {
```

Comment:
We ended up with two copies of this method, one with the correct spelling and one with the incorrect spelling.